src/Common/src/System/SR.cs

```diff
@@ -32,6 +32,11 @@ private static bool UsingResourceKeys()
             return false;
         }
 
+        internal static string GetResourceString(string resourceKey)
+        {
+            return GetResourceString(resourceKey, resourceKey);
```

Comment:
(I didn't realise at first that it was included via an `<Import/>`, which was why I missed it)